### PR TITLE
Expand German Locazion

### PR DESCRIPTION
**Pull request by Sprungente**
and small fix and improve :D

### CODE REVIEW

**Comment by montlikadani** [comment]
More changes? I will only accept if there are more than 5 edits.
[Read here](https://github.com/Zrips/Jobs/wiki/Translations) to learn more about editing the language.

**Comment by Sprungente** [comment]
it will get more :)

File: src/main/resources/locale/messages_de.yml [hunk 1]
```diff
@@ -296,7 +359,13 @@ command:
       info: 'Alle Jobs verlernen.'
     error:
       nojobs: 'Du hast keine Jobs!'
-    success: 'Du hast keine Jobs mehr.'
+    success: 'Du hast keine Jobs mehr.'$
```
Comment (montlikadani) [inline src/main/resources/locale/messages_de.yml]: Hmm.. $ moni,
Delete

Comment (Sprungente) [reply src/main/resources/locale/messages_de.yml]: you see everything 🥇

**Comment by montlikadani** [comment]
Want more changes? If not, I'll merge it.

**Comment by Sprungente** [comment]
You can merge 😋

Am 24.11.2018 um 19:42 schrieb montlikadani <notifications@github.com<mailto:notifications@github.com>>:


Want more changes? If not, I'll merge it.

**Pull request merged**